### PR TITLE
Add PriceListItem aggregate and impletement "create price list item" usecase

### PR DESCRIPTION
**Pull request by AndreiGanichev**
Also add CategoryCode attribute to PriceListITem table

### CODE REVIEW

File: src/Modules/Payments/Application/PriceListItems/CreatePriceListItem/CreatePriceListItemCommand.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using CompanyName.MyMeetings.Modules.Payments.Application.Configuration.Commands;
+using Newtonsoft.Json;
+
+namespace CompanyName.MyMeetings.Modules.Payments.Application.PriceListItems.CreatePriceListItem
+{
+    public class CreatePriceListItemCommand : InternalCommandBase<Guid>
```
Comment (kgrzybek) [inline src/Modules/Payments/Application/PriceListItems/CreatePriceListItem/CreatePriceListItemCommand.cs:7]: This is not Internal Command. It is normal Command (invoked by User through API) so it should inherit from `CommandBase<Guid>`

File: src/Modules/Payments/Application/PriceListItems/CreatePriceListItem/CreatePriceListItemCommand.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+        public string SubscriptionPeriodCode { get; }
+        
+        public string CategoryCode { get; }
+        
+        public decimal PriceValue { get; }
+
+        public string PriceCurrency { get; }
+
+        [JsonConstructor]
```
Comment (kgrzybek) [inline src/Modules/Payments/Application/PriceListItems/CreatePriceListItem/CreatePriceListItemCommand.cs:19]: To remove, only needed for internal commands

File: src/Modules/Payments/Domain/PriceListItems/PriceListItem.cs [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            var priceListItem = new PriceListItem();
+            
+            var priceListItemCreatedEvent = new PriceListItemCreatedDomainEvent(
+                Guid.NewGuid(), 
+                countryCode,
+                subscriptionPeriod.Code,
+                category.Code,
+                price.Value,
+                price.Currency,
+                isActive: false);
```
Comment (kgrzybek) [inline src/Modules/Payments/Domain/PriceListItems/PriceListItem.cs:41]: In my opinion, it should be active by default.

File: src/Modules/Payments/Application/PriceListItems/CreatePriceListItem/CreatePriceListItemCommand.cs (hunk 2 above)
Comment (AndreiGanichev) [reply src/Modules/Payments/Application/PriceListItems/CreatePriceListItem/CreatePriceListItemCommand.cs]: Fixed

File: src/Modules/Payments/Application/PriceListItems/CreatePriceListItem/CreatePriceListItemCommand.cs (hunk 1 above)
Comment (AndreiGanichev) [reply src/Modules/Payments/Application/PriceListItems/CreatePriceListItem/CreatePriceListItemCommand.cs]: Fixed

File: src/Modules/Payments/Domain/PriceListItems/PriceListItem.cs (hunk 3 above)
Comment (AndreiGanichev) [reply src/Modules/Payments/Domain/PriceListItems/PriceListItem.cs]: Ok, made it active. Actually i implemented it active by default at the begining. But then i looked again at commands in Miro and decided that create-change-activate-deacticate was a implying workflow.

Comment (kgrzybek) [reply src/Modules/Payments/Domain/PriceListItems/PriceListItem.cs]: I agree, It was not clear

**Pull request merged**